hassio/src/hassio-main.ts

```diff
@@ -72,7 +77,7 @@ export class HassioMain extends SupervisorBaseElement {
   }
 
   protected render() {
-    if (!this.supervisor || !this.hass) {
+    if (!this.supervisor.localize || !this.hass) {
```

Comment:
Why do we wait for localize? We should just start rendering?

Reply:
On reconnect, translations are not properly loaded (if you switch from SU to configuration and back)

Reply:
Don't we update when the translations are loaded?

Reply:
Yes, but no, there is some weird stuff happening, this small adjustment makes it work 100% of the times

Reply:
fund the issue, will adjust and update